README.md

```diff
@@ -10,25 +10,26 @@
 
 Prior to submitting your first milestone, please make sure that you have successfully completed a [KYB](https://in.sumsub.com/idensic/l/#/uni_LhZH9SdrmtuvGnnz) (or, if you work alone, a [KYC](https://in.sumsub.com/idensic/l/#/uni_ZUSbgdezo6hdrZVn)) check.
```

Comment:
I'm not sure it needs to be prior to the submission. Theoretically, we could also do the KYC/KYB process after.

Reply:
KYC/KYB is needed prior to milestone submission to assert that the team is somehow committed to participating in the JAM prize, especially because there is no pre-registration process to filter out bogus/spam applicants.

From the perspective of the Fellows, it is important not to spend time reviewing submissions from teams only to have them fail basic KYC/KYB checks later down the line.